esx_service/vsan_policy.py

```diff
@@ -67,6 +68,7 @@ def create_policy_file(filename, content):
             f.write(content)
             f.write('\n')
     except:
+        logging.exception("Failed to open %s for writing", filename)
```

Comment:
Added logging to find_child. I left setStatusDetached as-is since exception handler there is used to auto-correct an issue.